app/uk/gov/hmrc/tradergoodsprofilesdatastore/repositories/RecordsRepository.scala

```diff
@@ -69,4 +70,39 @@ class RecordsRepository @Inject() (
       )
       .toFuture()
       .map(_.getDeletedCount > 0)
+
+  def saveRecord(record: GoodsItemRecords): Future[Boolean] =
+    collection
+      .replaceOne(
+        filter = byRecordId(record.recordId),
+        replacement = record,
+        options = ReplaceOptions().upsert(true)
+      )
+      .toFuture()
+      .map(_ => true)
+
+  def update(eori: String, recordId: String, updateRequest: UpdateRecordRequest): Future[Option[GoodsItemRecords]] =
+    get(recordId).flatMap {
```

Comment:
`eori ` is unused here

Reply:
Thanks for spotting that @yohan-oc - updated it :)